### PR TITLE
Add telemetry message v3

### PR DESCRIPTION
**Pull request by pedrovsg**

### CODE REVIEW

File: cmd/monitoring/main.go [hunk 1]
```diff
@@ -41,7 +42,16 @@ func main() {
...
+	producer, err := monitoring.NewProducer(ctx, cfg.Kafka, cfg.Kafka.Topic)
+	if err != nil {
+		log.Fatalf("failed to create kafka producer with error: %v", err)
+	}
+
+	telSchema, err := schemaRegistry.EnsureSchema(fmt.Sprintf("%s-value", cfg.Kafka.TelemetryTopic), monitoring.TelemetryAvroSchema)
+	if err != nil {
+		log.Fatalf("failed to prepare telemetry_config_set schema with error: %v", err)
+	}
+	telemetryProducer, err := monitoring.NewProducer(ctx, cfg.Kafka, cfg.Kafka.TelemetryTopic)
```
Comment (topliceanu) [inline cmd/monitoring/main.go:54]: I find this namic confusing: both `tel` and `telemetry`. They are also not telemetry, telemetry is done in offchain reporting in the `chainlink` repo. How about using `configSetSchema` and `configSetSimplifiedSchema`?

File: pkg/monitoring/config.go [hunk 2]
```diff
@@ -194,6 +196,9 @@ func parseEnvVars(cfg *Config) {
 	if value, isPresent := os.LookupEnv("KAFKA_TOPIC"); isPresent {
 		cfg.Kafka.Topic = value
 	}
```
Comment (topliceanu) [inline pkg/monitoring/config.go:198]: Please rename this topic as well.

File: pkg/monitoring/feed_monitor.go [hunk 3]
```diff
@@ -62,59 +67,88 @@ func (f *feedMonitor) Start(ctx context.Context) {
...
+		f.solanaConfig.NetworkName)
+	isLateAnswer := time.Since(time.Unix(int64(envelope.Answer.Timestamp), 0)).Seconds() > float64(f.feedConfig.HeartbeatSec)
+	f.metrics.SetOffchainAggregatorAnswerStalled(isLateAnswer, f.feedConfig.ContractAddress.String(),
+		f.solanaConfig.ChainID, f.feedConfig.ContractStatus, f.feedConfig.ContractType,
+		f.feedConfig.FeedName, f.feedConfig.FeedPath, f.solanaConfig.NetworkID,
+		f.solanaConfig.NetworkName)
+	return nil
+}
+
+func (f *feedMonitor) processTelemetry(envelope StateEnvelope) error {
```
Comment (topliceanu) [inline pkg/monitoring/feed_monitor.go:139]: As I understand it, this is not telemetry, in the chainlink-ocr2 sense, no?! Telemetry goes into the implementation of `types.MonitoringEndpoint`. Please rename this helper

File: pkg/monitoring/mapping.go [hunk 4]
```diff
@@ -155,6 +156,60 @@ func MakeTransmissionMapping(
...
+	}
+	signers, err := convertToJson(extractSigners(state.Oracles))
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse signers: %w", err)
+	}
+	transmitters, err := convertToJson(extractTransmitters(state.Oracles))
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse transmitters: %w", err)
+	}
+	s, err := convertToJson(int32ArrToInt64Arr(offchainConfig.S))
```
Comment (topliceanu) [inline pkg/monitoring/mapping.go:181]: `convertToJson` is a alias of `encoding/json#Marshal`! Just use `Marshal`

File: pkg/monitoring/schemas.go [hunk 5]
```diff
@@ -160,10 +160,35 @@ const TransmissionAvroSchema = `
...
+    {"name": "delta_resend", "type": "bytes", "doc": "uint64 big endian"},
+    {"name": "delta_round", "type": "bytes", "doc": "uint64 big endian"},
+    {"name": "delta_grace", "type": "bytes", "doc": "uint64 big endian"},
+    {"name": "delta_stage", "type": "bytes", "doc": "uint64 big endian"},
+    {"name": "r_max", "type": "long", "doc": "uint32"},
+    {"name": "s", "type": "string"},
+    {"name": "oracles", "type": "string"},
+    {"name": "feed_contract_status", "type": "string"}
+  ]
+}
```
Comment (topliceanu) [inline pkg/monitoring/schemas.go:184]: The way we encode schemas has changed in the main branch.
I'll push and update on the main branch!

**Review by topliceanu — CHANGES_REQUESTED** [review]

File: pkg/monitoring/mapping.go [hunk 6]
```diff
@@ -0,0 +1,319 @@
...
+		"delta_resend":         uint64ToBeBytes(offchainConfig.DeltaResendNanoseconds),
+		"delta_round":          uint64ToBeBytes(offchainConfig.DeltaRoundNanoseconds),
+		"delta_grace":          uint64ToBeBytes(offchainConfig.DeltaGraceNanoseconds),
+		"delta_stage":          uint64ToBeBytes(offchainConfig.DeltaStageNanoseconds),
+		"delta_c":              uint64ToBeBytes(numericalMedianOffchainConfig.DeltaCNanoseconds),
+		"alpha_ppb":            uint64ToBeBytes(numericalMedianOffchainConfig.AlphaAcceptPpb),
+		"r_max":                int64(offchainConfig.RMax),
+		"s":                    s,
+		"oracles":              oracles,
+		"feed_contract_status": feedConfig.ContractStatus,
```
Comment (pedrovsg) [inline pkg/monitoring/mapping.go:207]: this is wrong, should be state_account

**Pull request closed without merging**